### PR TITLE
PLATFORM-2986: Script for verifying old SVG files

### PR DESCRIPTION
**Pull request by themech**

### CODE REVIEW

File: maintenance/checkSVGs.php [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+	public function checkFile( $file ) {
+		$this->mSVGNSError = false;
+		$check = new XmlTypeCheck(
+			$file,
+			[ $this, 'checkSvgScriptCallback' ],
+			[ 'processing_instruction_handler' => 'UploadBase::checkSvgPICallback' ]
+		);
+		if ( $check->wellFormed !== true ) {
+			return 'uploadinvalidxml';
+		} elseif ( $check->filterMatch ) {	// validation error
```
Comment (WikiaTech) [inline maintenance/checkSVGs.php]: INFO Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

File: maintenance/checkSVGs.php [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+class UploadDummy extends UploadBase {
+	public function initializeFromRequest( &$request ) {}
+
+	/**
+	 * @param String Full contents of svg file
+	 *
+	 * @return true if ok, or string for error code
+	 */
+	public function checkFile( $file ) {
```
Comment (WikiaTech) [inline maintenance/checkSVGs.php]: MAJOR Reduce the number of returns of this function 4, down to the maximum allowed 3. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

[INFO]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO'
[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

**Review by Grunny — APPROVED** [review]
LGTM.

**Pull request merged**